luminaire/utils/validation.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
```

Comment:
Any other file name suggestion for this? Since, we do not want just one file with all utility functions. 
Might be one file per utility type.